README.md

```diff
@@ -30,54 +28,57 @@ $ pip install meeshkan
 
 Note that `meeshkan` requires **Python 3.6+.**
 
-## Python API
+## Getting started with Meeshkan
 
-### Quick start
+The basic Meeshkan flow is **collect, build and mock.**
+1. To start, **collect** data from recorded server traffic and/or OpenAPI specs.
+1. Then, **build** a schema that unifies these various data sources.
+1. Finally, use this schema to create a **mock** server of an API.
 
-```python
-from meeshkan.schemabuilder.update_mode import UpdateMode
-from meeshkan.schemabuilder import build_schema_batch, build_schema_online, update_openapi
-from openapi_typed_2 import OpenAPIObject
-import meeshkan
-from typing import List
-import json
-from http_types import HttpExchange, Request, Response, ResponseBuilder, RequestBuilder, HttpMethod
+### Tutorial
 
+The [Meeshkan tutorial](https://github.com/meeshkan/meeshkan-tutorial) covers the most essential aspects of the collect, build and mock flow.
 
-def read_http_exchanges() -> List[HttpExchange]:
-    """Read HTTP exchanges from the source of your choice.
-    """
-    request: Request = RequestBuilder.from_url(
-        "https://example.org/v1", method=HttpMethod.GET, headers={})
+### Hello world
 
-    response: Response = ResponseBuilder.from_dict(dict(
-        statusCode=200, body=json.dumps({"hello": "world"}), headers={}))
-
-    http_exchange: HttpExchange = HttpExchange(request=request, response=response)
+```bash
+$ meeshkan record -r --daemon # start the recorder in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # record traffic
+$ meeshkan record --stop-daemon # stop the daemon
+$ meeshkan build
+$ meeshkan mock -r --daemon # start the mocking server in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # mock traffic
+$ meeshkan mock --stop-daemon # stop the daemon
+```
 
-    return [http_exchange]
+## Collect recordings of API traffic
 
+To build a Meeshkan spec, you must first **collect** recordings of server traffic and/or OpenAPI server specs.
```

Comment:
```suggestion
Let's look at how to build a Meeshkan spec. First, you have to **collect** recordings of server traffic and/or OpenAPI server specs.
```

The wording of this sentence and the following are a bit repetitive, so trying to switch it up a bit.